boilerplate/js/default/src/index.js

```diff
@@ -6,6 +6,7 @@ Blits.Launch(App, 'app', {
   h: 1080,
   multithreaded: false,
   debugLevel: 1,
+  focusHoldTimeout: 50,
```

Comment:
I would prefer to not add this setting to the boilerplate and let the defaults do their work, unless there is a specific need to adjust this timeout

Reply:
Done